src/assets/country_codes.json

```diff
@@ -0,0 +1,252 @@
+{
```

Comment:
It'd probably be easier to get these from an npm package. https://www.google.com/search?q=npm+package+country+codes&oq=npm+package+country+codes&aqs=chrome..69i57.2032j1j7&sourceid=chrome&ie=UTF-8 gives a few good ones; [`i18n-iso-countries`](https://www.npmjs.com/package/i18n-iso-countries) seems pretty popular.